plone/app/layout/viewlets/tests/test_content.py

```diff
@@ -78,6 +78,31 @@ def test_pub_date(self):
         self.assertEqual(viewlet.pub_date(), None)
 
 
+    def test_anonymous_users_see_byline_if_show_enabled(self):
+        self.security_settings.allow_anon_views_about = True
+        self.logout()
+        viewlet = self._get_viewlet()
+        self.assertTrue(viewlet.show())
+
+    def test_anonymous_users_dont_see_byline_if_show_disabled(self):
+        self.security_settings.allow_anon_views_about = False
+        self.logout()
+        viewlet = self._get_viewlet()
+        self.assertFalse(viewlet.show())
+
+    def test_logged_users_see_byline_if_show_enabled(self):
```

Comment:
I don't see the point on replicating these tests for authenticated users also; I think one test should be enough.